pkg/web/apps/apps_psql_repository.go

```diff
@@ -106,3 +110,58 @@ func (a *appsPostgreSQLRepository) GetAppVersionsByAppID(id string) (*[]models.V
 
 	return &versions, nil
 }
+
+// GetAppByID retrieves an app by id from the database
+func (a *appsPostgreSQLRepository) GetAppByID(ID string) (*models.App, error) {
+	var app models.App
+
+	sqlStatement := `SELECT id,app_id,app_name FROM app WHERE id=$1;`
+	row := a.db.QueryRow(sqlStatement, ID)
+	err := row.Scan(&app.ID, &app.AppID, &app.AppName)
+	if err != nil {
+		log.Error(err)
+		if err == sql.ErrNoRows {
+			return nil, models.ErrNotFound
+		}
+		return nil, models.ErrInternalServerError
+	}
+
+	return &app, nil
+
+}
+
+// UpdateAppVersions all versions sent
+func (a *appsPostgreSQLRepository) UpdateAppVersions(versions []models.Version) error {
```

Comment:
Should remove the `s` and be `UpdateAppVersionByID`

Reply:
The input is not the ID but a list of versions. In this way, shows not be a good idea to use ByID in this one. WDYT?

Reply:
Sorry, looked at the code wrong. Might be nice to abstract the loop to the service layer and have this just update a single version. It is not a big deal though.

Reply:
Please, give a look at how it is now and let me know what do you think.